### DISCUSSION TITLE
Github marketplace and status not having dark mode

### DISCUSSION THREAD

**Discussion by ChewSW28** [community/community#45701] [Accessibility]
### Select Topic Area

General

### Body

![image](https://user-images.githubusercontent.com/98095959/215320147-9701b85e-d305-4f02-a15f-6fa1a48ac802.png)
![image](https://user-images.githubusercontent.com/98095959/215320225-46bc4141-669b-4b24-b528-77486e1d1b31.png)
 
For refrence,this is how my Github homepage looks like:
![image](https://user-images.githubusercontent.com/98095959/215320380-a1bc34b2-38f3-4cea-8839-c08830553319.png)

**Comment by EUSTW** [comment]
New Solution

**Reply by ChewSW28** [reply]
What is it?

**Comment by luongvo** [comment]
👍

**Comment by ghostinhershell** [comment]
Hey there @HIFun-cws, thanks so much for reaching out! I do see that dark mode not working when in GitHub Marketplace is a documented issue, but I don't see an issue for this also being missing when viewing the GitHub Status page. I'll go ahead and file this with the proper team so that it can get some visibility. Thanks again!

**Reply by nyxb** [reply]
So I don't have darkmode on the marketplace either, is it not possible or how do I fix the error?

**Reply by Thomashighbaugh** [reply]
**not a GitHub employee** No one does it seems, @ghostinhershell is escalating the issue to the team that will eventually fix it but in the meantime you could always use `Stylus` (a browser extension) to find a stylesheet to give the GitHub marketplace a dark mode.

**Comment by cehfisher** [comment]
Hello @HIFun-cws - this issue appears to be resolved now. Could you please confirm and close out this issue? Thank you! (cc @queenofcorgis)
